docs/reference/versions.md

```diff
@@ -0,0 +1,43 @@
+# Charmed etcd versions
+
+Charmed etcd is shipped in the following [tracks](https://documentation.ubuntu.com/juju/3.6/reference/charm/#track): 
+
+* [Charmed etcd 3.6](https://charmhub.io/charmed-etcd?channel=3.6/beta) (channel `3.6/beta`)
+
+## etcd 3.6
+
+**Base:** Noble (Ubuntu 24.04)
+
+**Supported architectures:** `amd64` and `arm64`.
+
+### Supported features
+
+* Automated deployment on VM
```

Comment:
is this about Terraform? if yes, could you add a link to the repo terraform dir?